src/styles/index.scss

```diff
@@ -1,3 +1,77 @@
+$hand-transform-origin: center bottom;
+
+@mixin center {
+  display: flex;
```

Comment:
It is a good practice to add a fallback for older browsers that do not support flex. You can use a mixin for this purpose. For example: @mixin flexbox { display: -webkit-box; display: -moz-box; display: -ms-flexbox; display: -webkit-flex; display: flex; }.